core/templates/pages/topic-editor-page/editor-tab/topic-editor-tab.directive.ts

```diff
@@ -331,12 +331,18 @@ angular.module('oppia').directive('topicEditorTab', [
               newPracticeTabIsDisplayed) {
             if (
               newPracticeTabIsDisplayed !==
-              $scope.topic.getPracticeTabIsDisplayed()) {
+              $scope.topic.getPracticeTabIsDisplayed() && $scope.topicHasAtLeastTenPracticeQuestions()) {
```

Comment:
Let's not hardcode the minimum number in the name -- what if we want to change it later on. Let's instead call it something like `hasMininumPracticeQuestions` or similar.